prow/scripts/cluster-integration/kyma-gke-integration.sh

```diff
@@ -7,7 +7,7 @@
 #
 # - REPO_OWNER - Set up by prow, repository owner/organization
 # - REPO_NAME - Set up by prow, repository name
-# - PULL_NUMBER - Set up by prow, Pull request number
+# - BUILD_TYPE - Set up by prow, pr/master/release
```

Comment:
`BUILD_TYPE` is not set by prow

Reply:
Technically speaking it is set by the preset-labels which are managed by prow. We have `preset-build-master: "true"` and `preset-build-pr: "true"` which set `BUILD_TYPE` to either `pr/master/release`

Reply:
IMO we should mention that it is set by specyfing given preset in job definition.